blockchain/types/tx_internal_rlp_encode_test.go

```diff
@@ -266,6 +275,10 @@ func testTxRLPEncodeDynamicFee(t *testing.T) {
 	})
 	assert.Equal(t, nil, err)
 
+	assert.Equal(t, hash(txHashRLP.Bytes()), NewTx(tx).Hash())
+	senderTxHash, isFDTx := NewTx(tx).SenderTxHash()
```

Comment:
Sorry for make you confuse;; 
I think you can use `SenderTxHashAll` to check senderTxHash works.